nebula-algorithm/src/main/resources/application.conf

```diff
@@ -33,20 +33,22 @@
         weightCols: ["start_year"]
     }
 
-    # algo result sink into Nebula. If data.sink is nebula, then this nebula.write config can be valid.
-    write:{
-        # Nebula graphd server address， multiple addresses are split by English comma
+  # Nebula Graph reader by ngql
+  nebula-ngql: {
```

Comment:
The options of data.source should be `nebula,nebula_ngql,csv,text`

Reply:
Is `nebula-ngql` tag in in an appropriate location?
The `nebula` tag cannot be close by brace。

Reply:
Why delete the `nebula.write` ? 
I think `nebula-ngql` tag is an option of `data.source`.